lib/sandbox/postman-legacy-interface.js

```diff
@@ -93,24 +187,151 @@ _.assign(PostmanLegacyTestInterface.prototype, /** PostmanLegacyTestInterface.pr
      * @returns {Object}
      */
     getResponseCookie: function (cookieName) {
-        return _.isString(cookieName) ? this.responseCookies[cookieName.toLowerCase()] : undefined;
+        return _.isString(cookieName) ? this.__cookies[cookieName.toLowerCase()] : undefined;
     },
 
     /**
      * @param {String} headerName
      * @returns {String}
      */
     getResponseHeader: function (headerName) {
-        return _.isString(headerName) ? this.responseHeaders[headerName.toLowerCase()] : undefined;
+        var header = (this.__execution.response && this.__execution.response.headers) &&
+            this.__execution.response.headers.one(headerName);
+
+        return header ? header.value : undefined;
     }
 });
 
 module.exports = {
-    create: function (options) {
-        if (_.get(options, 'masked.scriptType') === 'test') {
-            return new PostmanLegacyTestInterface(options);
+    setup: function (scope, execution, options) {
+
+        /**
+         * @name SandboxGlobals
+         * @type {Object}
+         */
+        var globalvars = _.assign({}, scopeLibraries);
+
+        // set global variables that are exposed in legacy interface
+        // ---------------------------------------------------------
+
+        // 1. set the tests object (irrespective of target)
+        /**
+         * Store your assertions in this object
+         * @memberOf SandboxGlobals
+         * @type {Object}
+         */
+        globalvars.tests = _.isObject(execution.tests) ? execution.tests : (execution.tests = {});
+
+        // 2. set common environment, globals and data
+        /**
+         * All global variables at the initial stages when the script ran
+         * @memberOf SandboxGlobals
+         * @type {Object}
+         */
+        globalvars.globals = execution.globals.syncVariablesTo();
+
+        /**
+         * All environment variables at the initial stages when script ran
+         * @memberOf SandboxGlobals
+         * @type {Object}
+         */
+        globalvars.environment = execution.environment.syncVariablesTo();
+
+        /**
+         * The data object if it was passed during a collection run
+         * @memberOf SandboxGlobals
+         * @type {Object}
+         */
+        globalvars.data = execution.data || (execution.data = {});
+
+        // 3. set the request object in legacy structure
+        /**
+         * The request that will be sent (or has been sent)
+         * @memberOf SandboxGlobals
+         * @type {Object}
+         */
+        globalvars.request = execution.request ? {
+            id: execution.legacy ? execution.legacy._itemId : undefined,
+            name: execution.legacy ? execution.legacy._itemName : undefined,
+            description: execution.request.description ?
+                execution.request.description.toString() : undefined,
+            headers: execution.request.getHeaders(),
```

Comment:
Looks like pathVariables are not exposed in the sandbox, so we're good. From the Chrome App:

![image](https://cloud.githubusercontent.com/assets/3726484/24607105/da627352-188d-11e7-8724-6b9fbad82207.png)